app/serializers/task_serializer.rb

```diff
@@ -19,6 +19,10 @@ def lite_paper
     object.paper
   end
 
+  def is_metadata_task
```

Comment:
Kind of agree with this, but maybe there's a good reason to leave this as `is_metadata_task`?